spec/models/books_model_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  it do
```

Comment:
I personally think one-line statements look better too, but after reading this in the Wolox tech guide I chose not to use them:
"Use {} only when defining let or subject if fits in one line. For everything else use do; end instead of {}. This is because combining both styles in examples can be confusing when reading the code.

# bad
it {}

# good
it do
end"

What do you think? I'm pretty sure I should follow this example
Also, I'm now using shoulda-matchers.